kolibri/core/assets/src/navigation.vue

```diff
@@ -42,59 +21,14 @@
           title: 'Kolibri',
           home_link: '/',
         },
-        userNavDisplay: false,
```

Comment:
So the content navigation (sidebar?) is going to be a sub-element of the nav bar?